aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/SnapshotCommand.java

```diff
@@ -154,7 +160,37 @@ public Integer call() throws Exception {
                 snapshotName = String.format("Snapshot-%s", new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS").format(new Date()));
             }
 
-            // Run snapshot
+            //TODO: refactor after release to get separated workflows
+            if (app.isOnboarded()) {
+                log.info("Triggering snapshot for an application using Fast-Scan workflow.");
+                ScanAndReScanApplicationJobRequest.ScanAndReScanApplicationJobRequestBuilder requestBuilder = ScanAndReScanApplicationJobRequest.builder()
+                        .appGuid(applicationGuid);
+                String targetNode = app.getTargetNode();
+                if (StringUtils.isNotEmpty(targetNode)) {
+                    requestBuilder.targetNode(targetNode);
+                }
+                String caipVersion = apiInfoDto.getApiVersion();
+                if (StringUtils.isNotEmpty(caipVersion)) {
```

Comment:
Isn't CAIP version a information from the application rather than from aip console ? I mean it doesn't look like a user parameter so maybe we can let the node set that value itself ?

Reply:
Node selection is not an option of snapshot command